assets/app/view/spreadsheet.rb

```diff
@@ -120,7 +120,13 @@ def render_corporation(corporation)
       h(:tr, props, [
         h(:th, corporation_color, corporation.name),
         *@game.players.map do |p|
-          h(:td, p.num_shares_of(corporation).to_s + (corporation.president?(p) ? '*' : ''))
+          style_props =
+            if (@game.round.is_a? Engine::Round::Stock) && @game.round.players_sold[p][corporation] == :prev
```

Comment:
add parens around method calls

Reply:
let's not do this, instead let's add a method to game base that tells you when something was sold, then you don't need to check if it's a stock round

something like, game.round.did_sell?(p, corporation)